Utilities/PlayerUtils.cs

```diff
@@ -322,6 +322,16 @@ public static bool CheckSolidGround(this Player player, int solidGroundAhead = 0
             }
             return ConditionMet;
         }
+
+        /// <summary>
+        /// Disables the default wing flap sound from vanilla; this is specifically to address issues with (currently) <see cref="CalamityMod.Items.Accessories.Wings.AureateBooster"/> and <see cref="CalamityMod.Items.Accessories.Wings.MOAB"/>.
+        /// </summary>
+        /// <param name="player">The Player to disable the sound on</param>
```

Comment:
No reason to name each case in the summary, just makes it so when we inevitably add a new case or remove a case it becomes silently outdated.